content/.gitbook/assets/getpull.json

```diff
@@ -0,0 +1,169 @@
+                      "type": "string",
+                      "description": "Status of the request",
+                      "example":"Success"
+
+                    },
+                    "database_owner": {
+                      "type": "string",
+                      "description": "Owner of the database",
+                      "example":"dolthub"
+                    },
+                    "database_name": {
+                      "type": "string",
+                      "description": "Database name",
+                      "example":"museum-collections"
+                    },
+                    "pull_id": {
+                      "type": "string",
+                      "description": "Pull request ID",
+                      "example":"1"
+                    },
+                    "title:": {
+                      "type": "string",
+                      "description": "Title of the pull request",
+                      "example":"Added new data"
+                    },
+                    "description:": {
+                      "type": "string",
+                      "description": "Description of the pull request",
+                      "example":"Added new data"
+                    },
+                    "state":{
+                      "type": "string",
+                      "description": "State of the pull request",
+                      "example":"merged"
+                    },
+                    "from_branch_owner":{
+                      "type": "string",
+                      "description": "Owner of the branch from which the pull request was created",
+                      "example":"liuliu"
+                    },
+                    "from_branch_database":{
+                      "type": "string",
+                      "description": "Database name of the branch from which the pull request was created",
+                      "example":"museum-collections"
+                    },
+                    "from_branch_name":{
+                      "type": "string",
+                      "description": "Name of the branch from which the pull request was created",
+                      "example":"feature"
+                    },
+                    "to_branch_owner":{
+                      "type": "string",
+                      "description": "Owner of the branch to which the pull request was created",
+                      "example":"dolthub"
+                    },
+                    "to_branch_database":{
+                      "type": "string",
+                      "description": "Database name of the branch to which the pull request was created",
+                      "example":"museum-collections"
+                    },
+                    "to_branch_name":{
+                      "type": "string",
+                      "description": "Name of the branch to which the pull request was created",
+                      "example":"main"
+                    },
```

Comment:
These descriptions are a little unclear. There needs to be a difference in the descriptions for `from_x` vs `to_x`